app/scripts/util.js

```diff
@@ -85,6 +103,7 @@ var util = {
 		}
 		page.redirect('/error');
 	},
+	emailRegex: '[a-z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&\'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?',
```

Comment:
You're right, but I'd rather they do something wrong and then get a warning for it than just not being allowed to write stupid stuff and not understanding why. Remember, this form will be mostly used by non-technical people.